iron-selectable.html

```diff
@@ -275,8 +305,8 @@
     },
 
     _updateAttrForSelected: function() {
-      if (this._shouldUpdateSelection) {
-        this.selected = this._indexToValue(this.indexOf(this.selectedItem));
+      if (this.selectedItem) {
```

Comment:
`selectedItem` should be a reference to the node itself, rather than its index (`selected`, as you mentioned above).